com.unity.render-pipelines.universal/Runtime/Passes/DrawObjectsPass.cs

```diff
@@ -125,9 +123,6 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                     : new Vector4(flipSign, 0.0f, 1.0f, 1.0f);
                 cmd.SetGlobalVector(ShaderPropertyId.scaleBiasRt, scaleBias);
 
-                context.ExecuteCommandBuffer(cmd);
-                cmd.Clear();
```

Comment:
This cannot be removed as well due to the RenderingUtils.RenderObjectsWithError.